test/integration/targets/azure_rm_acr/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+ - name: Create an ACR
+   azure_rm_acr:
+     name: "acr{{ resource_group | hash('md5') | truncate(7, True, '') }}"
```

Comment:
this won't work with ansible integration test framework (unless you just create your own resource group here).
ansible test framework provides two existing resource groups, and "resource_group" is a variable containing location of that resource group.

Reply:
as discussed, no fix required